source/gameengine/Physics/Bullet/CcdPhysicsController.h

```diff
@@ -768,6 +768,16 @@ class CcdPhysicsController : public PHY_IPhysicsController
 		m_cci.m_radius = margin;
 	}
 
+	// Cone Shape (radius & height)
+	virtual void SetConeShape(float radius, float height)
+	{
+		if (m_collisionShape && (m_collisionShape->getShapeType() == CONE_SHAPE_PROXYTYPE)) {
+			btConeShape *coneShape = static_cast<btConeShape *>(m_collisionShape);
+			coneShape->setRadius(radius);
```

Comment:
Bullet support well the shape changes ? No AABB to update or collision pair to remove ?